### PR TITLE
c1 cleanup and reorg

### PR DESCRIPTION
**Pull request by jmanico**
c1 cleanup and reorg

Moved most c1 requirements that are process, documentation or legal issues - or any requirement that a security verifier cannot verify - was moved to the governance appendix.

### CODE REVIEW

File: 1.0/en/0x10-C01-Training-Data-Governance.md [hunk 1]
```diff
@@ -13,232 +13,65 @@ Maintain a verifiable inventory of all datasets, accept only trusted sources, an
...
-| **1.3.3** | **Verify that** post-training fairness metrics are evaluated and documented. | 2 | D/V |
-| **1.3.4** | **Verify that** a lifecycle bias-management policy assigns owners and review cadence. | 3 | D/V |
+| **1.2.2** | **Verify that** all access to training data is logged, including user, time, and action. | 2 | D/V |
+| **1.2.3** | **Verify that** training datasets are encrypted in transit and at rest, using industry-standard cryptographic algorithms and key management practices. | 2 | D/V |
+| **1.2.4** | **Verify that** cryptographic hashes or digital signatures are used to ensure data integrity during training data storage and transfer. | 2 | D/V |
+| **1.2.5** | **Verify that** that automated anomaly detection techniques are applied to guard against unauthorized modifications or corruption of training data. | 2 | D/V |
```
Comment (ottosulin) [inline 1.0/en/0x10-C01-Training-Data-Governance.md]: I would leave out the "anomaly detection" from here completely, or if we want to have it specifically, separate it to another level 3 requirement. I would think _some kind of automated detection_ even if not super fancy is enough for level 2.

In general not easy to implement and I would prefer to keep level 2 still quite achievable.

File: 1.0/en/0x10-C01-Training-Data-Governance.md [hunk 2]
```diff
@@ -13,232 +13,65 @@ Maintain a verifiable inventory of all datasets, accept only trusted sources, an
...
+| **1.2.6** | **Verify that** all training dataset versions are uniquely identified, stored immutably, and auditable to support rollback and forensic analysis. | 3 | D/V |
```
Comment (ottosulin) [inline 1.0/en/0x10-C01-Training-Data-Governance.md]: Btw: you can do this now or later: need to order the requirements to follow the levels.

File: 1.0/en/0x10-C01-Training-Data-Governance.md [hunk 3]
```diff
@@ -13,232 +13,65 @@ Maintain a verifiable inventory of all datasets, accept only trusted sources, an
...
-| **1.7.1** | **Verify that** deletion workflows purge primary and derived data and assess model impact, and that the impact on affected models is assessed and, if necessary, addressed (e.g., through retraining or recalibration). | 1 | D/V |
-| **1.7.2** | **Verify that** mechanisms are in place to track and respect the scope and status of user consent (and withdrawals) for data used in training, and that consent is validated before data is incorporated into new training processes or significant model updates. | 2 | D |
-| **1.7.3** | **Verify that** workflows are tested annually and logged. | 2 | V |
+| **1.3.1** | **Verify that** cryptographic hashes or digital signatures are applied to label artifacts to ensure their integrity and authenticity. | 2 | D/V |
+| **1.3.2** | **Verify that** labeling interfaces and platforms enforce strong access controls, maintain tamper-evident audit logs of all labeling activities, and protect against unauthorized modifications. | 2 | D/V |
+| **1.3.3** | **Verify that** sensitive information in labels is redacted, pseudonymized, anonymized, or encrypted at rest and in transit.| 2 | D/V |
```
Comment (ottosulin) [inline 1.0/en/0x10-C01-Training-Data-Governance.md]: Is it possible that a reader could assume that "or encrypted at rest and in transit." could covered by just "storage bucket in GCP is already encrypted and I'm using TLS" which is kind of generic expected security control anyway vs. actually protecting the label itself?

I agree that this sort of application or database column/row encryption is along the same lines of control type pseudonymization, but obviously not nearly as good privacy control as anonymization.

What do you think about rephrasing the control to refer in some way to encryption of that piece of data so that it would not be confused with disk encryption & TLS?

File: 1.0/en/0x10-C01-Training-Data-Governance.md [hunk 4]
```diff
@@ -13,232 +13,65 @@ Maintain a verifiable inventory of all datasets, accept only trusted sources, an
...
-
-| # | Description | Level | Role |
-|:--------:|---------------------------------------------------------------------------------------------------------------------|:---:|:---:|
-| **1.9.1** | **Verify that** appropriate defenses, such as adversarial training (using generated adversarial examples), data augmentation with perturbed inputs, or robust optimization techniques, are implemented and tuned for relevant models based on risk assessment. | 3 | D/V |
-| **1.9.2** | **Verify that** if adversarial training is used, the generation, management, and versioning of adversarial datasets are documented and controlled. | 2 | D/V |
-| **1.9.3** | **Verify that** the impact of adversarial robustness training on model performance (against both clean and adversarial inputs) and fairness metrics is evaluated, documented, and monitored. | 3 | D/V |
-| **1.9.4** | **Verify that** strategies for adversarial training and robustness are periodically reviewed and updated to counter evolving adversarial attack techniques.| 3 | D/V |
+| **1.4.1** | **Verify that** automated tests catch format errors, nulls, and label skews on every ingest or significant data transformation. | 1 | D |
```
Comment (ottosulin) [inline 1.0/en/0x10-C01-Training-Data-Governance.md]: Should label skews be moved as a separate level 2 or even 3 control?

What is the basic thing everyone should be able to do? At least I think the format errors and nulls are reasonable requirement for everyone to ensure some level of quality.

File: 1.0/en/0x10-C01-Training-Data-Governance.md (hunk 1 above)
Comment (jmanico) [reply 1.0/en/0x10-C01-Training-Data-Governance.md]: So I am just going to drop the word anomaly and we get:

| **1.2.5** | **Verify that** that automated detection techniques are applied to guard against unauthorized modifications or corruption of training data. | 2 | D/V |

File: 1.0/en/0x10-C01-Training-Data-Governance.md (hunk 2 above)
Comment (jmanico) [reply 1.0/en/0x10-C01-Training-Data-Governance.md]: I'll do it now, good call

File: 1.0/en/0x10-C01-Training-Data-Governance.md (hunk 4 above)
Comment (jmanico) [reply 1.0/en/0x10-C01-Training-Data-Governance.md]: Good call, here is the whole section now:

## C1.4 Training Data Quality and Security Assurance

Combine automated validation, manual spot-checks, and logged remediation to guarantee dataset reliability.

| # | Description | Level | Role |
|:--------:|---------------------------------------------------------------------------------------------------------------------|:---:|:---:|

| **1.4.1** | **Verify that** automated tests catch format errors amd nulls on every ingest or significant data transformation. | 1 | D |
| **1.4.2** | **Verify that** LLM training and fine-tuning pipelines implement poisoning detection & data integrity validation (e.g., statistical methods, outlier detection, embedding analysis) to identify potential poisoning attacks (e.g., label flipping, backdoor trigger insertion, role-switching commands, influential instance attacks) or unintentional data corruption in training data. | 2 | D/V |
| **1.4.3** | **Verify that** appropriate defenses, such as adversarial training (using generated adversarial examples), data augmentation with perturbed inputs, or robust optimization techniques, are implemented and tuned for relevant models based on risk assessment. | 3 | D/V |
| **1.4.4** | **Verify that** automatically generated labels (e.g., via LLMs or weak supervision) are subject to confidence thresholds and consistency checks to detect hallucinated, misleading, or low-confidence labels. | 2 | D/V |
| **1.4.5** | **Verify that** automated tests catch label skews on every ingest or significant data transformation. | 3 | D |
